taxonomies/ingredients.txt

```diff
@@ -65821,7 +65884,7 @@ fr:extraits naturels
 <en:extract
 en:vegetable extract, vegetable extracts
 de:Pflanzenextrakt, Pflanzenextrakte, Gemüseextrakte
-es:Extracto vegetal
+es:Extracto vegetal, extratos vegetales
```

Comment:
is it extracto or extrato ? the singular is different than the plural

Reply:
Yes it is - I just added the plural to what was there but that was wrong.. I'll change it